pep-0723.rst

```diff
@@ -156,14 +157,19 @@ version of ``pyproject.toml``, which is called ``Cargo.toml``:
         println!("Did our date match? {}", re.is_match("2014-01-01"));
     }
 
-One important thing to note is that the metadata is embedded in a comment
-mostly for introspection since Rust documentation is generated from comments.
-Another is that users rarely edit dependencies manually, but rather use their
-Cargo package manager.
+One important thing to note is that the metadata is embedded in a doc-comment.
```

Comment:
Is there a link for what Rust means by a "doc-comment", as distinct from a normal comment? If so, we should include it so as not to assume familiarity with Rust on the part of the reader.